server/app-engine/static/gtfs-rt-util.js

```diff
@@ -89,6 +89,63 @@ if (!fetch) {
         return d;
     }
 
+
+    // returns 0 for Monday...and 6 for Sunday
+    exports.getDayOfWeek = function() {
+        if (testDow) {
+            return testDow;
+        } else return ((new Date()).getDay() + 6) % 7;
+    }
+
+    // returns date as an 8-character string (ie 20220317 for 3/17/22)
+    exports.getDate = function() {
```

Comment:
gtfs-rt-util.js already has a getDate() function (that does something different). Choose a different name, or else one function will be permanently hidden.